app/client/src/components/pages/Community.Tabs.DrinkingWater.js

```diff
@@ -191,7 +191,11 @@ function createAccordionItem(
             value: item.pws_type,
           },
           {
-            label: 'Drinking Water Health Based Violations',
+            label: (
+              <GlossaryTerm term="Drinking Water Health-based Violations">
+                Drinking Water Health Based Violations
```

Comment:
I think this should be hyphenated like it is in the glossary,
```suggestion
                Drinking Water Health-Based Violations
```
or
```suggestion
                Drinking Water Health-based Violations
```

Reply:
Yeah. I'm not sure why it's different. We did the same thing on the national page. I'll update it in both places and check the national page tests.